xdp-loader/README.org

```diff
@@ -69,6 +69,12 @@ Specify which ELF section to load the XDP program(s) from in each file. The
 default is to use the first program in each file. If this option is set, it
 applies to all programs being loaded.
 
+** -n, --prog-name <prog_name>
+Specify which BPF program with the name to load the XDP program(s) from in
+each file. The default is to use the first program in each file. Only one
+of --section and --prog-name may be specified, as both options are exclusive.
```

Comment:
"as both options are exclusive" is just restating "only one of... may be specified" - pick one :)